### PR TITLE
Add dotenv

### PR DESCRIPTION
**Pull request by Jack-Barry**
Hey amigo,

I really appreciated your tutorial. I did run into some issues though, and created this pull request in the hopes that others trying to build similar projects won't have some of the same troubles.

What was performed:

* Fixed some typos that resulted in faulty building and generating (these should definitely be fixed in the tutorial code, if nothing else)
* Added the ability to use `.env` file during development to keep secret keys more secure
* Threw in some `async` functions to make things easier to follow

What changes for the developer:

* You can set environment variables via command line or in the admin panel of tools like Netlify (no more API keys in code! 🙌🏻)
* The `generate` and `build` commands *do* require setting `PRISMIC_API_URL` and `PRISMIC_API_KEY` since they are production builds. You can use a `.env` file when running in development mode though.

Let me know if this all makes sense, once again thanks for getting me on the right path to figure out this Nuxt thing!

Cheers 🍺

### CODE REVIEW

**Comment by kekkorider** [comment]
Hey @Jack-Barry, happy that my tutorial helped you!

Also, thank you for this PR, everything seems more than good; I will take a better look in the next days and get back to you.

Cheers!

**Comment by Jack-Barry** [comment]
🤦🏻‍♂️I just noticed a few issues when I deployed to Netlify with my personal project - will update this request when I get the code updated for it!

**Comment by Jack-Barry** [comment]
Without that last commit the pages would load on my Netlify deployment, but required refresh. It's all good now 👍🏼

File: nuxt.config.js [hunk 1]
```diff
@@ -1,116 +1,76 @@
-const pkg = require('./package')
-const Prismic = require('prismic-javascript')
-import { initApi } from './prismic.config'
+if (process.env.NODE_ENV !== "production") require("dotenv").config();
+const pkg = require("./package");
+const Prismic = require("prismic-javascript");
```
Comment (kekkorider) [inline nuxt.config.js]: This is not needed anymore.

File: nuxt.config.js [hunk 2]
```diff
@@ -1,116 +1,76 @@
...
+import { initApi, queryForDocType } from "./prismic.config";
```
Comment (kekkorider) [inline nuxt.config.js]: `initAPI` is not needed anymore.

File: pages/about.vue [hunk 3]
```diff
@@ -7,21 +7,19 @@
 
 <script>
 import Prismic from 'prismic-javascript'
```
Comment (kekkorider) [inline pages/about.vue]: This can be removed, it's not needed anymore.

File: pages/blog/index.vue [hunk 4]
```diff
@@ -15,26 +15,24 @@
 <script>
 import Prismic from 'prismic-javascript'
```
Comment (kekkorider) [inline pages/blog/index.vue]: This can be removed, it's not needed anymore.

File: pages/index.vue [hunk 5]
```diff
@@ -7,21 +7,19 @@
 
 <script>
 import Prismic from 'prismic-javascript'
```
Comment (kekkorider) [inline pages/index.vue]: This can be removed, it's not needed anymore.

**Review by kekkorider — CHANGES_REQUESTED** [review]
@Jack-Barry again, thanks a lot 🙏 

There are a couple of imports that should be removed since they're not needed anymore.

~Aso, I'm getting an error while running `nuxt generate` on my local machine (everything works fine when in development):~

EDIT: Never mind, I forgot to add `PRISMIC_API_URL ` and `PRISMIC_API_KEY` to the `generate` command.

Schermata 2019-06-14 alle 12 36 45

**Pull request closed without merging**